homeassistant/generated/config_flows_v2.json

```diff
@@ -0,0 +1,1417 @@
+    },
+    "elgato": {
+      "name": "Elgato Light"
+    },
+    "elkm1": {
+      "name": "Elk-M1 Control"
+    },
+    "elmax": {
+      "name": "Elmax"
+    },
+    "emonitor": {
+      "name": "SiteSage Emonitor"
+    },
+    "emulated_roku": {
+      "name": "Emulated Roku",
+      "translated_name": true
+    },
+    "enocean": {
+      "name": "EnOcean"
+    },
+    "enphase_envoy": {
+      "name": "Enphase Envoy"
+    },
+    "environment_canada": {
+      "name": "Environment Canada"
+    },
+    "epson": {
+      "name": "Epson"
+    },
+    "escea": {
+      "name": "Escea"
+    },
+    "esphome": {
+      "name": "ESPHome"
+    },
+    "evil_genius_labs": {
+      "name": "Evil Genius Labs"
+    },
+    "ezviz": {
+      "name": "Ezviz"
+    },
+    "faa_delays": {
+      "name": "FAA Delays"
+    },
+    "fibaro": {
+      "name": "Fibaro"
+    },
+    "filesize": {
+      "name": "File Size",
+      "translated_name": true
+    },
+    "fireservicerota": {
+      "name": "FireServiceRota"
+    },
+    "fivem": {
+      "name": "FiveM"
+    },
+    "fjaraskupan": {
+      "name": "Fj\u00e4r\u00e5skupan"
+    },
+    "flick_electric": {
+      "name": "Flick Electric"
+    },
+    "flipr": {
+      "name": "Flipr"
+    },
+    "flo": {
+      "name": "Flo"
+    },
+    "flume": {
+      "name": "Flume"
+    },
+    "flux_led": {
+      "name": "Magic Home"
+    },
+    "forecast_solar": {
+      "name": "Forecast.Solar"
+    },
+    "forked_daapd": {
+      "name": "forked-daapd"
+    },
+    "foscam": {
+      "name": "Foscam"
+    },
+    "freebox": {
+      "name": "Freebox"
+    },
+    "freedompro": {
+      "name": "Freedompro"
+    },
+    "fritz": {
+      "name": "AVM FRITZ!Box Tools"
+    },
+    "fritzbox": {
+      "name": "AVM FRITZ!SmartHome"
+    },
+    "fritzbox_callmonitor": {
+      "name": "AVM FRITZ!Box Call Monitor"
+    },
+    "fronius": {
+      "name": "Fronius"
+    },
+    "fully_kiosk": {
+      "name": "Fully Kiosk Browser"
+    },
+    "garages_amsterdam": {
+      "name": "Garages Amsterdam",
+      "translated_name": true
+    },
+    "gdacs": {
+      "name": "Global Disaster Alert and Coordination System (GDACS)"
+    },
+    "generic": {
+      "name": "Generic Camera"
+    },
+    "geocaching": {
+      "name": "Geocaching"
+    },
+    "geofency": {
+      "name": "Geofency"
+    },
+    "geonetnz_quakes": {
+      "name": "GeoNet NZ Quakes"
+    },
+    "geonetnz_volcano": {
+      "name": "GeoNet NZ Volcano"
+    },
+    "gios": {
+      "name": "GIO\u015a"
+    },
+    "github": {
+      "name": "GitHub"
+    },
+    "glances": {
+      "name": "Glances"
+    },
+    "goalzero": {
+      "name": "Goal Zero Yeti"
+    },
+    "gogogate2": {
+      "name": "Gogogate2 and ismartgate"
+    },
+    "goodwe": {
+      "name": "GoodWe Inverter"
+    },
+    "google": {
+      "name": "Google",
+      "integrations": {
+        "google": {
+          "name": "Google Calendars"
```

Comment:
It should be Google Calendar? CC @allenporter

Reply:
Yes, the product is called `Google Calendar`

Reply:
I assume I need to update homeassistant/components/google/manifest.json to fix this, is that right?

Reply:
yes